packages/backend/src/modules/tvl2/modules/Tvl2Module.ts

```diff
@@ -90,11 +93,28 @@ export function createTvl2Module(
   const statusRouter = createTvl2StatusRouter(config.tvl2, clock)
   const tvlRouter = createTvl2Router(tvlController, clock)
 
+  const tvlCleaner = new TvlCleaner(
+    clock,
+    logger,
+    syncOptimizer,
+    peripherals.getRepository(TvlCleanerRepository),
+    [
+      peripherals.getRepository(AmountRepository),
+      peripherals.getRepository(BlockTimestampRepository),
+      peripherals.getRepository(PriceRepository),
+      peripherals.getRepository(ValueRepository),
+    ],
+  )
+
   const start = async () => {
     await hourlyIndexer.start()
 
     await priceModule.start()
 
+    if (config.tvl2 && config.tvl2.tvlCleanerEnabled) {
```

Comment:
First condition is not needed, please also move this to the end of start function